rebar.config

```diff
@@ -82,6 +82,12 @@
         {emcl, {git, "https://github.com/aeternity/emcl.git",
                {ref, "007e406"}}},
 
+        {aeconnector, {git, "https://github.com/aeternity/aeconnector.git",
+          {branch, main}}},
+
+        {erlydtl, {git, "https://github.com/erlydtl/erlydtl.git",
```

Comment:
What is this lib for?

Reply:
I agree my friend it looks like ...But the reason behind is the need to support validators by minimal tooling when they start to manage a balance and review a commitments.

We have a such tool: https://github.com/aeternity/aeconnector/wiki/Telegram-bot

This is related to the commitments management and allows to render a reporting template which is to be send on attached channel.  

P.S. We may introduce something like regex instead and remove later (but we need a time)

Reply:
Ok, where to we use it?

Reply:
We use it in our BTC [connector](https://github.com/aeternity/aeconnector/blob/main/src/catalogue/aeconnector_btc_full_node.erl) (I hoped that we can list it on aeconnector list of deps but it requires to be part of aeternity deps otherwise code isn’t loaded)